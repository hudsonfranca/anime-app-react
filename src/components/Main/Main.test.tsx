import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { Main } from '.';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

let wrapper: ShallowWrapper;

beforeEach(() => {
  wrapper = shallow(<Main />);
});

describe('<Main/> redering', () => {
  it('Should render without errors', () => {
    const wrapper = renderer.create(<Main />).toJSON();

    expect(wrapper).toMatchSnapshot();
  });

  it('should render a <Container>', () => {
    expect(wrapper.find(`[data-test='container']`)).toHaveLength(1);
  });
});
