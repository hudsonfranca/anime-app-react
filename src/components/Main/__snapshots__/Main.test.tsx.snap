// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> redering Should render without errors 1`] = `
.c0 {
  grid-area: main;
  margin-top: 5px;
  margin-bottom: 5px;
  width: 100%;
  height: 100%;
}

<div
  className="c0"
  data-test="container"
/>
`;
